chore(tags): add macosarmjava tag (#226)
PrismLauncher/refraction: tags/macosarmjava.md

---

> title: Install Arm Java on MacOS

Reviewer: ```suggestion
title: Install arm64 Java on macOS
```

color: yellow
aliases: ['armmacosjava']
---

> On MacOS Arm, also known as Apple Silicon (M1, M2, etc.), you will need to install The Arm Version of Java. We recommend using Azul, the links for which can be found below:

Reviewer: ```suggestion
On macOS arm64, also known as Apple Silicon (M1, M2, etc.), you will need to install the arm64 version of Java. We recommend using builds from Azul, the links for which can be found below:
```

> - [Java 8](https://aka.ms/vs/17/release/vc_redist.x64.exe) for versions ≤ 1.16

Reviewer 2: 👀

Author: Oops

- [Java 17](https://www.azul.com/downloads/?version=java-17-lts&os=macos&architecture=arm-64-bit&package=jdk#zulu) for versions ≥ 1.17

> It is recommended to install both Java 8 and Java 17

Reviewer: ```suggestion
It is recommended to install both Java 8 and Java 17.
```
